packages/cli/src/test-utils/command_runner.ts

```diff
@@ -86,7 +87,11 @@ export class TestCommandRunner {
       // AsyncLocalStorage is used to make sure that we're capturing outputs only from the same asynchronous context
       // in potentially concurrent environment.
       await asyncLocalStorage.run(interceptor, async () => {
-        await this.parser.parseAsync(args);
+        await this.parser
+          .parseAsync(args)
+          .then(async (argv) =>
+            this.usageDataEmitter.emitSuccess({}, { command: argv._.join(' ') })
```

Comment:
I don't think the test command runner should invoke the usageDataEmitter. We should have tests to ensure that the entry point is wiring up the usageDataEmitter properly, but there's no reason for all of the tests to be invoking this noop